apps/frontend/src/components/WorkoutMenu/WorkoutEditor.tsx

```diff
@@ -288,12 +288,15 @@ export const WorkoutEditor = ({
       </Text>
       <FormControl>
         <HStack>
-          {canSaveRemotely ? (
-            <Button onClick={saveRemotely} leftIcon={<Icon as={CloudUpload} />}>
+          {canSaveRemotely && (
```

Comment:
just moved from 
`canSaveRemotely ? <COMP /> : null`
to
`canSaveRemotely && <Comp />`

Can ofc change back (and change the new button).
I just think that it is easier to read without that trailing null
But open for other pros/cons

Reply:
Since `canSaveRemotely` is a `boolean`, it is more or less the same. One could argue that `&&` have some gotchas with how JSX is rendered, aka the falsey value `0` is rendered as `0`, and some people have just decided to only use ternaries for conditional rendering. I kinda get that point, and think it could make sense to keep all the view code kinda the same, but yeah 🤷

Reply:
makes sense, classic js. i can change it back 🚀